tutorials/xr/setting_up_xr.rst

```diff
@@ -89,6 +90,24 @@ Next we need to add a script to our root node. Add the following code into this
 		else:
 			print("OpenXR not initialised, please check if your headset is connected")
 
+  .. code-tab:: csharp
+
+    public partial class XRInitializer : Node3D
```

Comment:
Since the root node in the scene is called `Main`, I think this class should use the same name.

```suggestion
    public partial class Main : Node3D
```